digitizer/plot.py

```diff
@@ -103,7 +104,7 @@ def get_bokeh_plot(self, isotherm_dict):
 
     def on_click_download(self):
         """Download JSON file."""
-        return self.isotherm.json_str
+        return StringIO(self.isotherm.json_str)
```

Comment:
Ok, so this works for you now?
I remember that I played around with this a bit - I'll need to check whether your code works on the server as well.

Reply:
Please do check the server instance. The download option wasn't working on my local instance and this solution was identified in the documentation.

Reply:
Thanks, I checked and it seems to work.